{{cookiecutter.project_slug}}/frontend/src/views/main/admin/AdminUsers.vue

```diff
@@ -8,76 +8,74 @@
       <v-btn color="primary" to="/main/admin/users/create">Create User</v-btn>
     </v-toolbar>
     <v-data-table :headers="headers" :items="users">
-      <template slot="items" slot-scope="props">
-        <td>{{ props.item.name }}</td>
-        <td>{{ props.item.email }}</td>
-        <td>{{ props.item.full_name }}</td>
-        <td><v-icon v-if="props.item.is_active">checkmark</v-icon></td>
-        <td><v-icon v-if="props.item.is_superuser">checkmark</v-icon></td>
-        <td class="justify-center layout px-0">
-          <v-tooltip top>
-            <span>Edit</span>
-            <v-btn slot="activator" flat :to="{name: 'main-admin-users-edit', params: {id: props.item.id}}">
-              <v-icon>edit</v-icon>
-            </v-btn>
-          </v-tooltip>
-        </td>
+      <template v-slot:item.is_active="{ item }">
```

Comment:
My editor (VS-Code) gives me the following error: `'v-slot' directive doesn't support any modifier`. I googled this and found the following answer: https://stackoverflow.com/a/63309643
Which solved this "issue" for me.

Except of this project I never programmed in JS, therefore I don't know the exact reason and if this solution is appropriate. However, I changed my code to look like this (I hate red warnings 😛):
```vue
    <v-data-table :headers="headers" :items="users">
      <template v-slot:[`item.is_active`]="{ item }">
        <v-icon v-if="item.is_active">mdi-check</v-icon>
      </template>

      <template v-slot:[`item.is_superuser`]="{ item }">
        <v-icon v-if="item.is_superuser">mdi-check</v-icon>
      </template>

      <template v-slot:[`item.actions`]="{ item }">
        <v-icon @click="routeEditUser(item.id)">mdi-pencil</v-icon>
      </template>
```

Reply:
What version of eslint are you using?
VScode probably complains because either the language server is unhappy or a linter emits a warning. In this case I think it is the latter (eslint) and the warning you are describing must be a new default, because I did not see this before.
From my understanding of Vue this is indeed [valid syntax](https://vuejs.org/v2/guide/components-slots.html#Dynamic-Slot-Names) and would fix the warning. However, I think it is kind of ugly in this case and maybe suppressing the warning (either via inline comment or global setting) would be better?

Reply:
My version is `v6.8.0` and indeed, the "correct" syntax isn't that beautiful. I'll provide a request change with the corresponding ignore comment.

Reply:
Seems like others run into this problem as well: https://github.com/vuejs/eslint-plugin-vue/issues/260#issuecomment-668794653

Reply:
Have you tried `<!-- eslint-disable-next-line eslint-plugin-vue/valid-v-slot -->`

Reply:
Ok, I checked if a update of everything to the latest version fixes this and it did not. *But* I also opened the project in PyCharm Professional and the error was not linted at all. So it seems to me, that this is a problem of VS-Code? Anyway, as this problem is not present in all editors I wouldn't change it.

Reply:
Try running `eslint` directly on the file (via command-line) and see if the error appears. Correctly configured editors will generally output the same warnings as they are just displaying what either the vue language server or (in our case) eslint say. I think the error you are seeing is some new default rule (ie. vue/valid-v-slot) that eslint-plugin-vue complains about and there is an bug in the plugin that prohibits correctly disabling it.